Tests/varLib/instancer_test.py

```diff
@@ -1916,6 +1917,212 @@ def test_normalizeAxisLimits_missing_from_fvar(varfont):
         instancer.normalizeAxisLimits(varfont, {"ZZZZ": 1000})
 
 
+def _get_name_records(varfont):
+    nametable = varfont["name"]
+    return {
+        (r.nameID, r.platformID, r.platEncID, r.langID): r.toUnicode()
+        for r in nametable.names
+    }
+
+
+def test_updateNameTable_with_registered_axes(varfont):
+    # Regular
+    instancer.updateNameTable(varfont, {"wght": 400})
+    names = _get_name_records(varfont)
+    assert names[(1, 3, 1, 0x409)] == "Test Variable Font"
+    assert names[(2, 3, 1, 0x0409)] == "Regular"
+    assert names[(3, 3, 1, 0x0409)] == "2.001;GOOG;TestVariableFont-Regular"
+    assert names[(6, 3, 1, 0x409)] == "TestVariableFont-Regular"
+    assert (16, 3, 1, 0x409) not in names
+    assert (17, 3, 1, 0x409) not in names
+
+    # Black
```

Comment:
splitting into separate unit test methods would be preferable. You could even parametrize the test input/expected output using `pytest.mark.parameterize` decorator, so you only write the test code once and only change the inputs and expected results. But that's optional.

Reply:
Thanks. I'll parameterize my tests.